_unittest/test_02_2D_modeler.py

```diff
@@ -135,12 +135,12 @@ def test_07_create_ellipse(self):
         assert ellipse1.solve_inside
         assert ellipse1.model
         assert ellipse1.material_name == "vacuum"
-        assert isclose(ellipse2.faces[0].area, math.pi * 4.0 * 4.0 * 3, rel_tol=0.1)
+        assert isclose(ellipse2.faces[0].area, math.pi * 4.0 * 4.0 * 3, relative_tolerance=0.1)
```

Comment:
maybe just tolerance?

Reply:
@maxcapodi78 the fact is that the method isclose accepts two different tolerances: relative and absolute.